### DISCUSSION TITLE
Going prone

### DISCUSSION THREAD

**Discussion by Quartofel** [FakeFishGames/Barotrauma#7564] [Ideas and suggestions]
Pretty self-explainatory, adding the ability for the characters to go a bit more personal with the floor.

It would would make room breaching outside of the water with more people easier and more enjoyable as well as unlock some new sub design options without relying on some clunky and questionable practices like slamming a hatch into character's jaw to "compress" them.

**Comment by MaaiG** [comment]
- Besides, you can always pretend to be dead. Lying among the corpses you can set up an ambush. ))
- You can get out of the line of fire (or friendly fire).

P.S. https://github.com/Regalis11/Barotrauma/discussions/6655
